### DISCUSSION TITLE
modifying chktxt to not pass some warning messages

### DISCUSSION THREAD

**Discussion by madecunningly** [latex-lsp/texlab#886] [Q&A]
I'm using the texlab setting to run chktex on open and save. However, I'd really like to modify the chktex to ignore a couple warnings. If I was using chktex directly, I would call chktex with something like 
`chktex --localrc myrcfile mytexfile.tex`  (myrcfile being the resource file modifying the chktex warnings)
However, as chktex is being called by texlab, how do I configure texlab to call chktex with the flag for the local resource file?

**Comment by pfoerster** [comment]
@madecunningly Thanks for the suggestion! With the next version, you will be able to use `texlab.chktex.additionalArgs` to pass extra arguments to ChkTeX.
